lib/jekyll/document.rb

```diff
@@ -39,7 +41,7 @@ def initialize(path, relations = {})
       end
 
       data.default_proc = proc do |_, key|
-        site.frontmatter_defaults.find(relative_path, collection.label, key)
+        site.frontmatter_defaults.find(relative_path, type, key)
```

Comment:
Was `collection.label` already a symbol? If not, is our replacing it with a symbol here safe?

Reply:
Yes, it is safe because, it is expected (from the comments):
https://github.com/jekyll/jekyll/blob/b82e4ea9203257deb991d4c9ea7d60e3e3848fc5/lib/jekyll/frontmatter_defaults.rb#L51-L70